tests/src/test/java/alluxio/shell/AlluxioShellUtilsTest.java

```diff
@@ -76,10 +77,23 @@ public String resetFileHierarchy() throws IOException, AlluxioException {
     return resetFileHierarchy(mFileSystem);
   }
 
+  /**
+   * Reset the file hierarchy.
```

Comment:
We usually use the active voice for method javadoc, so use "Resets" instead of "Reset" here and a few other places